### DISCUSSION TITLE
Get the request host attribute in controller while using DTOs

### DISCUSSION THREAD

**Discussion by Evoolo** [symfony/symfony#52644] [Q&A]
Hello,

I'm using DTOs to validate and passing request parameters to my controller. But I need the subdomain of the host attribute of an request in some functions. What is the best way to get this value while not loading the whole request object into my function?

**Comment by ghost** [comment]
Sorry but you have to rewrite discussion's description, maybe provide same code samples. Current one is not understandable.
